### ISSUE TITLE
About training the squeezenet on mnist

### ISSUE THREAD

**Issue by xuzhongyou** [pytorch/vision#901]
i try to train squeezenet on mnist dataset, but the loss has not changed. Can u show me the strategy about training.

**Comment by pmeier** [comment]
We  have not much to go on from the information you have given us. You could start with this [basic tutorial](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#sphx-glr-beginner-blitz-cifar10-tutorial-py) and successively change the dataset, the classifier, etc. to fit your needs. Otherwise see this [example](https://github.com/pytorch/examples/tree/master/mnist) for MNIST training.

**Comment by xuzhongyou** [comment]
i meant that i train the mnist dataset with the official squeezenet, but the loss almost unchanges, i do not know whether the learning strategy, i use sgd(lr = 0.02,mon=0.9). However when i use the batchnormal, the loss can reduce for training. so what's the problem of the official squeezenet, can u show the squeezenet training code not the squeezenet network.




------------------ 原始邮件 ------------------
发件人: "Philip Meier"<notifications@github.com>; 
发送时间: 2019年5月12日(星期天) 下午4:40
收件人: "pytorch/vision"<vision@noreply.github.com>; 
抄送: "徐忠友"<936214756@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [pytorch/vision] About training the squeezenet on mnist (#901)




We  have not much to go on from the information you have given us. You could start with this basic tutorial and successively change the dataset, the classifier, etc. to fit your needs. Otherwise see this example for MNIST training.

**Comment by pmeier** [comment]
The weights for `SqueezeNet` provided in the `torchvision` package are trained with [this script](https://github.com/pytorch/examples/tree/master/imagenet). In the future they will be trained [by this](https://github.com/pytorch/vision/blob/master/references/classification/train.py).

Please note that `SqueezeNet` is supposed to be fed by `224x224x3` images while MNIST provides `28x28x1` images. Thus, in its original form `SqueezeNet` is probably to deep for these small images to get a reasonable results.

**Comment by xuzhongyou** [comment]
Thank u very much for u answer, i think i have known the problem.




------------------ 原始邮件 ------------------
发件人: "Philip Meier"<notifications@github.com>; 
发送时间: 2019年5月12日(星期天) 下午5:23
收件人: "pytorch/vision"<vision@noreply.github.com>; 
抄送: "徐忠友"<936214756@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [pytorch/vision] About training the squeezenet on mnist (#901)




The weights for SqueezeNet provided in the torchvision package are trained with this script. In the future they will be trained by this.
 
Please note that SqueezeNet is supposed to be fed by 224x224x3 images while MNIST provides 28x28x1 images. Thus, in its original form SqueezeNet is probably to deep for these small images to get a reasonable results.

**Comment by datumbox** [comment]
This issue appears to be stale, so I'm closing it. If you think it should remain open, feel free to reopen it.
